scripts/generators/foundry.py

```diff
@@ -1,20 +1,21 @@
-import yaml
 from glob import glob
 import json
 import os
-import pandas as pd
-import re
 from shutil import copy
 import fire
-import time
 import datetime
 
-secrets = json.load(open('../../secrets.json'))
+# Moves with fields that are problematic for the export
+# - Non-moves such as "Run Away" are excluded
+# - Struggle and Growl have attributes like "Strength/special"
+# - Copycat Damage1 is "Same as the copied move"
+# - Simple Beam is rolled with Empathy. This skill exists neither on the Pokémon sheet nor on the trainer sheet (?)
```

Comment:
Empathy is Allure for Trainers, the trainer sheets don't have labeled correctly. it comes up alot, lol

Reply:
So the attribute roll should just be for Allure instead? I'm confused, even if it's mislabeled on the trainer sheet, there's still no Empathy on Pokémon

Reply:
Pokemon have Allure, Trainers have Empathy. Neither has both. (Empathy is understanding pokemon speak, which pokemon needless to say can do, and Allure is making Humans understand you, which humans can do already)